### DISCUSSION TITLE
See raw zigbee messages

### DISCUSSION THREAD

**Discussion by tomasz-grobelny** [Koenkk/zigbee2mqtt#10854] [Devices]
Hi all,

Is there any way to see raw zigbee messages from device? I have an issue described here: https://github.com/Koenkk/zigbee2mqtt/issues/10785 and the thing is that the device seems to be supported, but in practice it does not get temperature/humidity reports and I am trying to figure out what is going on.

**Comment by Koenkk** [comment]
I expect that if the device sends messages it should be logged already.

You can [sniff](https://www.zigbee2mqtt.io/advanced/zigbee/04_sniff_zigbee_traffic.html#with-cc2531) the Zigbee traffic to see if the device actually sends something.
